lib/gui/models/supported-formats.js

```diff
@@ -128,7 +128,25 @@ SupportedFormats.service('SupportedFormatsModel', function() {
       return false;
     }
 
-    return this.isSupportedImage(path.basename(image, `.${extension}`));
+    return this.isSupportedImage(path.basename(imagePath, `.${extension}`));
+  };
+
+  /**
+   * @summary Check if an image seems to be a Windows image
+   * @function
+   * @public
+   *
+   * @param {String} imagePath - image path
+   * @returns {Boolean} whether the image seems to be a Windows image
+   *
+   * @example
+   * if (SupportedFormatsModel.looksLikeWindowsImage('path/to/en_windows_7_ultimate_with_sp1_x86_dvd_u_677460.iso')) {
+   *   console.log('Looks like a Windows image');
+   * }
+   */
+  this.looksLikeWindowsImage = (imagePath) => {
+    const re = /windows|win7|win8|win10|winxp/i;
```

Comment:
Can we rename this variable to `regex` or something like that, just to make it ridiculously easy to the eye?